hw/top_earlgrey/dv/verilator/chip_sim_tb.cc

```diff
@@ -46,7 +46,7 @@ int main(int argc, char **argv) {
   // to appropriately propagate.
   // The reset duration must be appropriately sized to the divider for clk_aon
   // in chip_earlgrey_verilator.sv.  It must be at least 2 cycles of clk_aon.
-  simctrl.SetInitialResetDelay(500);
+  simctrl.SetInitialResetDelay(10000);
```

Comment:
@rswarbrick 
@vogelpi 
@imphil 

sorry my verilator / cpp skills suck..but is there an easy way to make verilator probe an internal signal instead of manually doing delays like this?

Reply:
Not really: lots of Verilator's optimisations depend on the fact that arbitrary internal signals aren't visible. I think the usual way to do feedback like this is to add (possibly using bind) a bit of SV code that does a DPI callback when the event you're interested in happens.

Reply:
ah okay, i can look into that eventually then by following some of the examples here.